config/settings/development.yml

```diff
@@ -23,3 +23,7 @@ zip_endpoints:
     access_key_id: 'foo'
     secret_access_key: 'bar'
 workflow_services_url: 'https://sul-lyberservices-test.stanford.edu/workflow/'
+
+slow_queries:
+  enable: true
```

Comment:
i wonder if we do want this on by default - how often will we have prescat running dev mode without caring about this, vs. actually caring about this?  but I don't feel strongly.